packages/libs/src/sdk/api/challenges/ChallengesApi.ts

```diff
@@ -70,14 +69,18 @@ export class ChallengesApi extends GeneratedChallengesApi {
       case ChallengeId.MOBILE_INSTALL:
       case ChallengeId.SEND_FIRST_TIP:
       case ChallengeId.TRACK_UPLOADS:
-        return `${args.userId}`
+        return `${encodeHashId(args.userId)}`
```

Comment:
Just curious:
IIRC, `encodeHashId` is not _guaranteed_ to produce output and can return `null` if the input is not a valid ID. What seems like it would generate some pretty subtle nested failures if it were to happen here. Any thoughts on using a zod parser or something that will throw if the input is invalid so that we get a clearer error in those cases?